src/process_collector.rs

```diff
@@ -27,13 +25,13 @@ const METRICS_NUMBER: usize = 7;
 pub struct ProcessCollector {
     pid: pid_t,
     descs: Vec<Desc>,
-    cpu_total: Mutex<Counter>,
-    open_fds: Gauge,
-    max_fds: Gauge,
-    vsize: Gauge,
-    rss: Gauge,
-    start_time: Gauge,
-    threads: Gauge,
+    cpu_total: IntCounter,
```

Comment:
I totally get the performance argument, but could we please keep the `cpu_total` metric as `f64`? All the other metrics (file descriptor counts, memory usage, threads) are actual integers, but processes that don't use a lot of CPU will now have 1 second CPU usage spikes every so often instead of a more or less flat usage with actual spikes occurring when they actually do.

E.g. a process using 1% CPU on average, with a 10% spike every minute will now look as if it is using 0% CPU most of the time and spike every 2 minutes out of 3.

I really don't think the marginal performance improvement is worth it. Please keep using `Counter` for `cpu_total`.